src/store/actions.js

```diff
@@ -1,70 +1,123 @@
 export default {
-  _setState({ commit }, data) {
-    commit('setState', data);
-  },
-  _getInitialData({ dispatch }) {
-    if (localStorage.getItem('appLists')) {
-      dispatch('_setInitialState', JSON.parse(localStorage.getItem('appLists')));
-    } else {
-      dispatch('_addList', {
-        name: 'default',
-        id: null,
-        items: [],
-        filters: {
-          tags: [],
-          categories: [],
-        },
-        checkedFilters: {
-          tags: [],
-          categories: [],
-        },
-      });
-    }
-  },
+  // local storage
 
-  // lists
+  _setListIdFromLocalStorage({ commit, dispatch }) {
+    const currentListId = JSON.parse(localStorage.getItem('currentListId'));
 
-  _setInitialState({ commit }, lists) {
-    commit('setInitialState', lists);
+    if (currentListId) {
+      commit('setCurrentListId', currentListId);
+      dispatch('_fetchListById', currentListId);
+    }
   },
-  _addList({ commit, dispatch }, list) {
-    commit('addList', list);
-
-    dispatch('_updateLocal');
+  _updateListIdInLocalStorage({ getters }) {
+    localStorage.setItem('currentListId', JSON.stringify(getters.currentListId));
```

Comment:
and here as well. I think we don't need it according that we work just with literal value (Number, String)